web/problems/models.py

```diff
@@ -49,6 +49,10 @@ def user_attempts(self, user):
     def user_solutions(self, user):
         return {attempt.part.id: attempt.solution for attempt in self.user_attempts(user)}
 
+    @property
```

Comment:
Načeloma koda spreminja le template za Octave/Matlab. Edino sprememba v `models.py`, je dodana metoda `slug`, ampak to ne bi smelo imeti vpliva na ostale programske jezike.

Reply:
Super, hvala!